src/ripple/app/paths/tests/Path_test.cpp

```diff
@@ -146,27 +146,68 @@ class Path_test : public beast::unit_test::suite
-            {USD}, Account("bob")["USD"](1));
-        Json::Value jv;
-        Json::Reader().parse(R"([{
-                "paths_canonical" : [],
-                "paths_computed" : [],
-                "source_amount" :
-                {
-                    "currency" : "USD",
-                    "issuer" : "r9QxhA9RghPZBbUchA9HkrmLKaWvkLXU29",
-                    "value" : "1"
-                }
-            }])", jv);
-        expect(jv == alternatives);
+        {
+            Env env(*this);
+            env.fund(XRP(10000), "alice", "bob", "carol",
+                "dan", "edward");
+            env.trust(Account("alice")["USD"](10), "bob");
+            env.trust(Account("bob")["USD"](10), "carol");
+            env.trust(Account("carol")["USD"](10), "edward");
+            env.trust(Account("alice")["USD"](100), "dan");
+            env.trust(Account("dan")["USD"](100), "edward");
+
+            auto const alternatives = findPath(env.open(), "alice", "edward",
+                {Account("alice")["USD"]}, Account("edward")["USD"](-1));
+            Json::Value jv;
+            Json::Reader().parse(R"([{
+                    "paths_canonical":[],
+                    "paths_computed":[
+                        [
+                            {
+                                "account":"rJ85Mok8YRNxSo7NnxKGrPuk29uAeZQqwZ",
+                                "type":1,
+                                "type_hex":"0000000000000001"
+                            },
+                            {
+					            "destination_amount" :
+					            {
+						            "currency" : "USD",
+						            "issuer" : "rNAnHhommqNJvV3mLieHADNJU6mfdRqXKp",
+						            "value" : "100"
+					            }
+                            }
+                        ],
+                        [
+                            {
+                                "account":"rPMh7Pi9ct699iZUTWaytJUoHcJ7cgyziK",
+                                "type":1,
+                                "type_hex":"0000000000000001"
+                            },
+                            {
+                                "account":"rH4KEcG9dEwGwpn6AyoWK9cZPLL4RLSmWW",
+                                "type":1,
+                                "type_hex":"0000000000000001"
+                            },
+                            {
+					            "destination_amount" :
+					            {
+						            "currency" : "USD",
+						            "issuer" : "rNAnHhommqNJvV3mLieHADNJU6mfdRqXKp",
+						            "value" : "10"
+					            }
+                            }
+                        ]
+                    ],
+                    "source_amount":{
+                        "currency":"USD",
+                        "issuer":"rG1QQv2nh2gr7RCZ1P8YYcBUKCCN633jCn",
+                        "value":"-1"
+                    }
+                }])", jv);
+
+            expect(jv == alternatives);
```

Comment:
Nice test. It would be moderately more obvious what's going on if you could replace the hard coded ripple addresses in `jv` with the account names, or some readable expression involving the account names.